src/styles.scss

```diff
@@ -0,0 +1,9 @@
+@import '~angular-material/angular-material.scss';
+
+body {
+  height: 100%;
+}
+body {
+  margin: 0;
+  font-family: Roboto, 'Helvetica Neue', sans-serif;
+}
```

Comment:
Why isn't this combined?

Reply:
Actually, this seems to still not be resolved.